### ISSUE TITLE
[18.10] CSS Enhancement

### ISSUE THREAD

**Issue by lolokai** [centreon/centreon#6928]
**Centreon Web version**: 18.10

**Centreon Engine version**:  18.10

**Centreon Broker version**:  18.10

**OS**: RHEL 7

**Additional environment details (AWS, VirtualBox, physical, etc.):**
VM

**Steps to reproduce the issue:**
The left menu is perfect but often when you use a light screen you can confound The 2nd level of menu and the 3rd level of menu

**Describe the results you expected:**
I made some modifications on the CSS in order to test a "black" theme and I think it's more user friendly :). Maybe you can add it into Centreon ?

![image](https://user-images.githubusercontent.com/2960490/47797319-62bf4c80-dd26-11e8-8a85-c145406fc4d4.png)

**Additional information you think important (e.g. issue happens only occasionally):**

**Comment by Rico29** [comment]
I totally agree with 'you can confound The 2nd level of menu and the 3rd level of menu'.

Can you please share your css ?
Regards

**Comment by lolokai** [comment]
I made only modifications using Inspector tools of my browser : sorry :/
